secret/secrets_manager_client.go

```diff
@@ -211,6 +211,32 @@ func (c *BasicSecretsManagerClient) UpdateSecretValue(ctx context.Context, in *s
 	return out, nil
 }
 
+// TagResource tags an existing secret.
+func (c *BasicSecretsManagerClient) TagResource(ctx context.Context, in *secretsmanager.TagResourceInput) (*secretsmanager.TagResourceOutput, error) {
+	if err := c.setup(); err != nil {
+		return nil, errors.Wrap(err, "setting up client")
+	}
+
+	var out *secretsmanager.TagResourceOutput
+	var err error
+	msg := awsutil.MakeAPILogMessage("TagResource", in)
+	if err := utility.Retry(
```

Comment:
I don't think we need := here

Reply:
I wrote a comment below for this, it's a stylistic preference.